app/components/polls/statements/CardButton.tsx

```diff
@@ -33,9 +58,16 @@ export const CardButton = <C extends ChoiceEnum | number>({
         typeof choice === "string" ? getButtonSize(choice) : false,
         choiceText && "rounded-md px-3 py-1 text-sm aspect-auto w-full mb-3",
         highlight && "bg-zinc-600",
+        extraButtonStyles,
       )}
+      onMouseEnter={() => setIsHovering(true)}
+      onMouseOut={() => setIsHovering(false)}
```

Comment:
(If it can be done without javascript then it almost always should be done without javascript)

Remove the hovering state and these handlers. Instead add a data attribute to store the active state `data-active={choice === activeChoice}`,
change the styles to be applied always based on the choice
```
[styles.agree]: choice === "agree",
[styles.disagree]: choice === "disagree",
[styles.skip]: choice === "skip",
```

and then change the styles to be applied on hover or when the data-active is true:

```css
.agree[data-active=true], .agree:hover { ...
```
This way we have less react state and fewer writes to the DOM (because we're no longer adding/removing classes when active state changes)

Reply:
Alright. On it.